dist/models/event.d.ts

```diff
@@ -30,6 +30,7 @@ declare namespace Event {
         maxParticipants?: number;
         recurring?: number;
         blockCalendar?: boolean;
+        displayInFeed?: boolean;
```

Comment:
@kcoleman731 why do we need this again? I heard an ex you gave was lunch, but why wouldn't that be part of the resturaunt calendar instead of the club?

Reply:
@tayhalla I think we need to allow admins to specify. If you look at Broadmoors cals, there are lots of events that probably don't belong in the feed.